### DISCUSSION TITLE
New Setup issue: The msg_id is too low, the client time has to be synchronized.

### DISCUSSION THREAD

**Discussion by srijaysunil** [Dineshkarthik/telegram_media_downloader#445] [Q&A]
Hi I am getting the below error when trying to run for the first time:

pyrogram.errors.BadMsgNotification: [16] The msg_id is too low, the client time has to be synchronized.

I have finished all the config and even authenticated with the phone verification.

**Comment by srijaysunil** [comment]
![Screenshot_82](https://github.com/Dineshkarthik/telegram_media_downloader/assets/22799837/1cd01d46-467b-452d-b4bb-962442fee029)

The screenshot shows how my system time(CST) is in sync with what the UTC time would be.

**Reply by ldrom** [reply]
Same issue here. I verified if I delete the session file and relogin as simulating a new run, it works perfectly for this single time. Upon trying the 2nd time, this "low msg_id" comes up again. I dont think it has something to do with time sync, as i tried to sync my macbook with 3 different servers and problem persists...

**Reply by srijaysunil** [reply]
Where is the session file located?

**Comment by dzwzqlx** [comment]
I think this is a new issue, I use this tool about once per two days, and this error happened just yestoday.

**Comment by dzwzqlx** [comment]
the owner's solution: 
https://github.com/Dineshkarthik/telegram_media_downloader/issues/446#issuecomment-1595644384
